pkg/proxy/nftables/proxier.go

```diff
@@ -1801,21 +1829,20 @@ func (proxier *Proxier) syncProxyRules() {
 				}
 			}
 		}
-	} else if !knftables.IsNotFound(err) {
+	} else {
 		proxier.logger.Error(err, "Failed to list nftables chains: stale chains will not be deleted")
```

Comment:
You already logged an error above. You can just remove the `if existingChains != nil {` check and the error here, and go straight into `for chain := range existingChains {`, which will just end up being a no-op if existingChains is nil.

Reply:
I wanted to preserve the "stale chains will not be deleted" part, but wasn't sure if it makes sense in the first error handling part. I guess you are fine with just removing it